dataprofiler/labelers/character_level_cnn_model.py

```diff
@@ -875,11 +874,9 @@ def _validate_training(self, val_data, batch_size_test=32,
                 x_val, batch_size=batch_size_test, verbose=verbose_keras)[1])
             y_val_test.append(np.argmax(y_val, axis=-1))
             batch_id += 1
-            sys.stdout.flush()
             if verbose_log:
-                sys.stdout.write(
-                    "\rEPOCH %g, validation_batch_id %d" % (
-                        self._epoch_id, batch_id))
+                logger.info("\rEPOCH %g, validation_batch_id %d"
+                            % (self._epoch_id, batch_id))
```

Comment:
previously, this would count as a single line, but this looks like it would create a new line per. Is there a way this would stream in?

Reply:
This is still just one line? I am confused I didn't change the content of the string message, I don't think I understand your question.

Reply:
In console it wouldn't create a new line for each write, but keep the counter increasing in place